.gitignore

```diff
@@ -14,8 +32,8 @@ rerun.txt
 pickle-email-*.html
 
 # TODO Comment out these rules if you are OK with secrets being uploaded to the repo
-config/initializers/secret_token.rb
-config/secrets.yml
+# config/initializers/secret_token.rb
+# config/secrets.yml
 
```

Comment:
You generally want to have the secret tokens in .gitignore. 
Your repos are public repos, which means everyone can read them. This also means that anyone can get access to this secret_token if it is included in git. This can open you up to being easily hacked.
In order to make this work though you will need to share these files that aren't in git with anyone on your team so that they can also work on the same application in production.